src/sagas/events.js

```diff
@@ -14,22 +14,38 @@ import { refreshEvents } from '../actions';
 
 function* createEvent({ eventData }) {
 
+    // let options = {
```

Comment:
Potential options.  Do we need any of these?

Reply:
I don't think so. Seems like they would only apply in use cases where specifics mattered more.